src/components/Camera/CameraTypes.js

```diff
@@ -12,23 +12,31 @@ type CameraCommonType = {
   onFailure: Function,
   onUserMedia: void => void,
   i18n: Object,
-  isFullScreen: boolean
+  isFullScreen: boolean,
 }
 
 type CameraActionType = {
   handleClick: Function,
   btnText: string,
   isFullScreen: boolean,
-  btnClass: string
+  btnClass: string,
+  btnDisabled: boolean,
+}
+
+type CameraErrorType = {
```

Comment:
I don't think it's a good idea to separate the type from the implementation, considering we are not importing it anywhere else apart from the class that is defined by it.